packages/collector/test/tracing/api/test.js

```diff
@@ -89,11 +86,51 @@ mochaSuiteFn('tracing/api', function () {
           expect(span.handleConstructorName).to.equal('SpanHandle');
         }));
 
+    it('must mark the current span as erroneous', () =>
+      controls
+        .sendRequest({
+          path: '/span/mark-as-erroneous'
+        })
+        .then(response => {
+          const span = response.span;
+          expect(span).to.exist;
+          expect(span.name).to.equal('node.http.server');
+          expect(span.errorCount).to.equal(1);
+          expect(span.data.http.error).to.equal(
+            'This call has been marked as erroneous via the Instana Node.js SDK, no error message has been supplied.'
+          );
+        }));
+
+    it('must mark the current span as erroneous with a custom error message', () =>
+      controls
+        .sendRequest({
+          path: '/span/mark-as-erroneous-custom-message'
+        })
+        .then(response => {
+          const span = response.span;
+          expect(span).to.exist;
+          expect(span.name).to.equal('node.http.server');
+          expect(span.errorCount).to.equal(1);
+          expect(span.data.http.error).to.not.exist;
+          expect(span.data.custom.path.error).to.equal('custom error message');
+        }));
+
+    it('must mark the current span as non-erroneous', () =>
+      controls
+        .sendRequest({
+          path: '/span/mark-as-non-erroneous'
+        })
+        .then(response => {
+          const span = response.span;
+          expect(span).to.exist;
+          expect(span.name).to.equal('node.http.server');
+          expect(span.data.http.error).to.not.exist;
```

Comment:
Q: Wondering if we also should assert for ec 0?

Reply:
Definitely. I accidentally dropped these assertions in the second commit. No idea why. Thanks for spotting this.